tooling/minimize/src/bb.rs

```diff
@@ -200,6 +200,57 @@ impl<'cx, 'tcx> FnCtxt<'cx, 'tcx> {
                         return Terminator::Goto(self.bb_name_map[&target.unwrap()]);
                     }
                 }
+                "raw_eq" => {
+                    let function_name = self.cx.get_fn_name(instance);
+                    if !self.functions.contains_key(function_name) {
```

Comment:
No need to synthesize a function here. `likely`/`unlikely` do that but it's kind of a hack.

If you make RawEq an intrinsic this will anyway be easier, you can just create a call to the intrinsic here.